draft-ietf-detnet-flow-information-model-09.xml

```diff
@@ -1159,7 +1066,10 @@
 <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
 <section anchor="sec_sum" title="Summary">
 
-        <t>This document describes DetNet flow information model and service information model for DetNet IP networks and DetNet MPLS networks. </t> 
+        <t>This document describes DetNet flow information model and service 
+		information model for DetNet IP networks and DetNet MPLS networks. 

```

Comment:
This document describes the DetNet flow information model and the service information model